quickstart.py

```diff
@@ -451,11 +453,19 @@ def print_results(request_id: str) -> None:
     print it to the console if so.
     """
     results_path = f"fides_uploads/{request_id}.json"
+
+    count = 0
+    max_allowed_waiting = 10
+    while not os.path.exists(results_path) or count > max_allowed_waiting:
+        logger.info("Waiting for privacy request results...")
+        time.sleep(5)
+        count += 1  # Only loop through a reasonable number of times
```

Comment:
could we print a friendly error in case the results don't get added  at all (after max wait time) for some reason?

Reply:
The existing script will raise a runtime error below on L472 if the results aren't found!